### PR TITLE
GitHub badges

### PR DESCRIPTION
**Pull request by proarafat**
I just add how to personalize the readme file by GitHub badges

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@proarafat 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Review by nazanin220737 — COMMENTED** [review]
+989058956231

File: content/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile.md [hunk 1]
```diff
@@ -230,6 +230,15 @@ If you authored any commit(s) present in the commit history for the listed tag o
 | [locationtech/jts](https://github.com/locationtech/jts) | 1.15.0 | [jts-1.15.0](https://github.com/locationtech/jts/releases/tag/jts-1.15.0) |
 | [apache/logging-log4j2](https://github.com/apache/logging-log4j2) | 2.11 | [log4j-2.11.0](https://github.com/apache/logging-log4j2/releases/tag/log4j-2.11.0) |
 
+#
+# 📛Add some badges In your readme profile
+#### There have many badges generator Website to generate the Badges for your Github Readme File
```
Comment (nazanin220737) [inline content/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile.md:235]:
```suggestion
#### There have many badges generator Website to generate the Badges for your Github Readme File
```

Comment (Bastek8) [reply content/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile.md:235]: > ```suggestion
> #### There have many badges generator Website to generate the Badges for your Github Readme File
> ```
>

**Comment by proarafat** [comment]
ok no problem

**Pull request closed without merging**